cli/medperf/commands/benchmark/run.py

```diff
@@ -0,0 +1,129 @@
+import medperf.config as config
+from medperf.commands.result.create import BenchmarkExecution
+from medperf.entities.result import Result
+from medperf.exceptions import ExecutionError, InvalidEntityError
+from medperf.utils import results_path
+from tabulate import tabulate
+
+
+class BenchmarkRunAll:
+    @classmethod
+    def run(
+        cls, benchmark_uid: str, data_uid: str, ignore_errors=False,
+    ):
+        """Runs all outstanding models of a benchmark
+        Args:
+            benchmark_uid:
+            data_uid: data uid
+        """
+        execution = cls(benchmark_uid, data_uid, ignore_errors)
+        execution.prepare()
+        execution.validate()
+        execution.prepare_cubes()
+        execution.run_models()
+        execution.print_summary()
+
+    def __init__(self, benchmark_uid: str, data_uid: str, ignore_errors=False):
+        self.benchmark_uid = benchmark_uid
+        self.data_uid = data_uid
+        self.model_execution = BenchmarkExecution(
+            benchmark_uid, data_uid, None, ignore_errors=ignore_errors,
+        )
+        self.outstanding_models = []
+        self.summary = {}
+
+    def prepare(self):
+        self.model_execution.prepare()
+        self.model_execution.model_uid = self.model_execution.benchmark.models[0]
+
+    def validate(self):
+        self.model_execution.validate()
+
+    def prepare_cubes(self):
+        self.model_execution.evaluator = self.model_execution.__get_cube(
+            self.model_execution.benchmark.evaluator, "Evaluator"
+        )
+        self.__filter_models()
+
+    def __filter_models(self):
+        benchmark_models = self.model_execution.benchmark.models
+        results = Result.all()
+        results = [
+            result
+            for result in results
+            if result.benchmark_uid == self.benchmark_uid
+            and result.dataset_uid == self.data_uid
+        ]
+        done_models = [result.model_uid for result in results]
+        self.outstanding_models = [
+            model for model in benchmark_models if model not in done_models
+        ]
+        self.summary["skipped_models"] = len(done_models)
+
+    def run_models(self):
+        self.summary["executions"] = []
+        for model in self.outstanding_models:
+            exec_summary = {
+                "model": model,
+                "success": False,
+                "partial": "",
+                "error": "",
+            }
+            # reset
+            self.model_execution.model_uid = model
+            self.model_execution.out_path = results_path(
```

Comment:
This is no longer supported. `results_path` is no longer an utils function. This also makes me wonder, can't we just create new `BenchmarkExecution`s for each model, and execute the `run` method here?